doc/source/whatsnew/v0.21.1.txt

```diff
@@ -56,6 +56,15 @@ Documentation Changes
 
 Bug Fixes
 ~~~~~~~~~
+- Bug in ``DataFrame.resample(...).apply(...)`` when there is a callable that returns different columns (:issue:`15169`)
+- Bug in :class:`TimedeltaIndex` subtraction could incorrectly overflow when ``NaT`` is present (:issue:`17791`)
+- Bug in :class:`DatetimeIndex` subtracting datetimelike from DatetimeIndex could fail to overflow (:issue:`18020`)
+- Bug in ``pd.Series.rolling.skew()`` and ``rolling.kurt()`` with all equal values has floating issue (:issue:`18044`)
+- Bug in ``pd.DataFrameGroupBy.count()`` when counting over a datetimelike column (:issue:`13393`)
+- Bug in ``pd.concat`` when empty and non-empty DataFrames or Series are concatenated (:issue:`18178` :issue:`18187`)
+- Bug in :class:`IntervalIndex` constructor when a list of intervals is passed with non-default ``closed`` (:issue:`18334`)
+- Bug in :meth:`IntervalIndex.copy` when copying and ``IntervalIndex`` with non-default ``closed`` (:issue:`18339`)
+- Bug in :func:``read_csv`` when reading numeric category fields with high cardinality (:issue:`18186`)
```

Comment:
Looks like all entries above yours got duplicated after rebasing #18408; you can see repeat entries under different sections below.  So, I think the following should be done:

- Delete all entries above yours (the repeat entries should still be there)
- Move your entry under either the I/O or Categorical section (not sure which is more appropriate, so should double check on this)
- When using references like `:func:` you only need single backticks, so <code>:func:\`\`read_csv\`\`</code> -> <code>:func:\`read_csv\`</code>

Reply:
Done. I put it under I/O because the direct call to CategoricalDType for the `expected` was not broken in the original code.